### PR TITLE
Added support for images in asset handler

### PR DESCRIPTION
**Pull request by dawnerd**
Figured this would make linking to images a lot easier. Doesn't return html, just the url.

### CODE REVIEW

File: helpers/asset.js [hunk 1]
```diff
@@ -15,6 +22,8 @@ module.exports = function (type, filename) {
     out = '<link rel="stylesheet" href="'+filePath+'"/>';
   } else if (type == 'js') {
     out = '<script src="'+filePath+'"></script>';
+  } else if (type == 'image') {
+    out = filePath;
```
Comment (jgallen23) [inline helpers/asset.js]: Should return image tag?

Comment (dawnerd) [inline helpers/asset.js]: Could, but I was thinking of a use case like `style="background: url({{asset "image" "someimage.jpg"}});"`

**Comment by jgallen23** [comment]
Maybe we Should have two types. Image and image-path.

**Comment by dawnerd** [comment]
Oh, I like that, I'll add it in.

**Pull request merged**